x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_view.tsx

```diff
@@ -0,0 +1,178 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiTitle,
+  EuiIcon,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { generateEncodedPath } from '../../../shared/encode_path_params';
+import { KibanaLogic } from '../../../shared/kibana';
+import { EuiButtonTo } from '../../../shared/react_router_helpers';
+import { COLLECTION_CREATION_PATH, COLLECTION_VIEW_PATH } from '../../routes';
+
+import { EnterpriseSearchAnalyticsPageTemplate } from '../layout/page_template';
+
+import { AnalyticsCollectionIntegrate } from './analytics_collection_integrate';
+import { AnalyticsCollectionSettings } from './analytics_collection_settings';
+
+import { FetchAnalyticsCollectionLogic } from './fetch_analytics_collection_logic';
+
+export const collectionViewBreadcrumbs = [
+  i18n.translate('xpack.enterpriseSearch.analytics.collectionsView.breadcrumb', {
+    defaultMessage: 'View collection',
+  }),
+];
+
+export const AnalyticsCollectionView: React.FC = () => {
+  const { fetchAnalyticsCollection } = useActions(FetchAnalyticsCollectionLogic);
+  const { analyticsCollection, isLoading, hasNoAnalyticsCollection } = useValues(
+    FetchAnalyticsCollectionLogic
+  );
+  const { name, section } = useParams() as { name: string; section: string };
+  const { navigateToUrl } = useValues(KibanaLogic);
+  const collectionViewTabs = [
+    {
+      id: 'events',
+      label: 'Events',
+      onClick: () =>
+        navigateToUrl(
+          generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name: analyticsCollection.name,
+            section: 'events',
+          })
+        ),
+      isSelected: section === 'events',
+    },
+    {
+      id: 'integrate',
+      label: 'Integrate',
+      prepend: <EuiIcon type="editorCodeBlock" size="l" />,
+      onClick: () =>
+        navigateToUrl(
+          generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name: analyticsCollection.name,
+            section: 'integrate',
+          })
+        ),
+      isSelected: section === 'integrate',
+    },
+    {
+      id: 'settings',
+      label: 'Settings',
+      onClick: () =>
+        navigateToUrl(
+          generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name: analyticsCollection.name,
+            section: 'settings',
+          })
+        ),
+      isSelected: section === 'settings',
+    },
+  ];
+
+  useEffect(() => {
+    fetchAnalyticsCollection(name);
+  }, []);
+
+  return (
+    <EnterpriseSearchAnalyticsPageTemplate
+      restrictWidth
+      isLoading={isLoading}
+      pageChrome={[...collectionViewBreadcrumbs]}
+      pageViewTelemetry="View Analytics Collection"
+      pageHeader={{
+        description: i18n.translate(
```

Comment:
change the page header to be more specific with the collection View page. Looks like a c&p of the collections list.